### PR TITLE
Docs: adjustments to Ubuntu install based on user feedback

### PR DESCRIPTION
**Pull request by chri2547**
This PR adds steps for installing prereq packages and updates the signing key command.

### CODE REVIEW

File: docs/sources/setup-grafana/installation/debian/index.md [hunk 1]
```diff
@@ -39,6 +39,19 @@ Grafana Enterprise is the recommended and default edition. It is available for f
...
+   ```
+
+1. Import the GPG key:
+
+   ```bash
+   sudo mkdir -p /etc/apt/keyrings/
+   wget -q -O - https://apt.grafana.com/gpg.key | gpg --dearmor | sudo tee /etc/apt/keyrings/grafana.gpg > /dev/null
+   ```
+
 1. To install required packages and download the Grafana repository signing key, run the following commands:
```
Comment (clayton-cornell) [inline docs/sources/setup-grafana/installation/debian/index.md:55]: Delete lines 55 through 62 - they are replaced by the added lines above.

Comment (chri2547) [inline docs/sources/setup-grafana/installation/debian/index.md:55]:
```suggestion
```

File: docs/sources/setup-grafana/installation/debian/index.md [hunk 2]
```diff
@@ -39,6 +39,19 @@ Grafana Enterprise is the recommended and default edition. It is available for f
...
 
    ```bash
```
Comment (chri2547) [inline docs/sources/setup-grafana/installation/debian/index.md:57]:
```suggestion
```

File: docs/sources/setup-grafana/installation/debian/index.md [hunk 3]
```diff
@@ -39,9 +39,20 @@ Grafana Enterprise is the recommended and default edition. It is available for f
...
+
+   ```bash
+   sudo mkdir -p /etc/apt/keyrings/
+   wget -q -O - https://apt.grafana.com/gpg.key | gpg --dearmor | sudo tee /etc/apt/keyrings/grafana.gpg > /dev/null
+   ```
+
+
    sudo apt-get install -y apt-transport-https
    sudo apt-get install -y software-properties-common wget
    sudo wget -q -O /usr/share/keyrings/grafana.key https://apt.grafana.com/gpg.key
```
Comment (chri2547) [inline docs/sources/setup-grafana/installation/debian/index.md:58]:
```suggestion
```

File: docs/sources/setup-grafana/installation/debian/index.md [hunk 4]
```diff
@@ -39,24 +39,32 @@ Grafana Enterprise is the recommended and default edition. It is available for f
...
+
+1. Import the GPG key:
+
+   ```bash
+   sudo mkdir -p /etc/apt/keyrings/
+   wget -q -O - https://apt.grafana.com/gpg.key | gpg --dearmor | sudo tee /etc/apt/keyrings/grafana.gpg > /dev/null
+   ```
+
+
    ```
```
Comment (chri2547) [inline docs/sources/setup-grafana/installation/debian/index.md:56]:
```suggestion
```

**Review by clayton-cornell — APPROVED** [review]
Looks good. Checked local doc build and it all looks correct. It matches the other Ubuntu docs, so.. good to go.

**Pull request merged**